tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers:
+#   proxy-httpd
+#   proxy-salt-broker
+#   proxy-squid
+#   proxy-ssh
+#   proxy-tftpd
+#
+# Maintainer: Maurizio Galli <maurizio.galli@suse.com>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
+
+    assert_script_run("podman pull $image");
```

Comment:
Depending on the size of the image you may want to add something like this (copied from [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/podman_network_cni.pm)

```suggestion
    assert_script_run("podman pull $image", timeout => 300, delay => 60, retry => 3););
```

Reply:
@ricardobranco777 you ment?
```suggestion
    script_retry("podman pull $image", timeout => 300, delay => 60, retry => 3);
```

Reply:
> @ricardobranco777 you ment?

Yes.  Thanks for pointing it out.

Reply:
One more thing, what does `script_retry` do exactly or where can I find the various definitions?

Reply:
> One more thing, what does `script_retry` do exactly or where can I find the various definitions? I found this http://open.qa/api/testapi/ however script_retry is missing

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L1846